File: JavaScript/printerErrors.js
const printerError = (s) => {
  let num = 0;
  for (let i = 0; i < s.length; i++) {
    if (!/[a-m]/.test(s[i])) {
      num++;
    }
  }
  return `${num}/${s.length}`;
};

console.log(printerErrors("aaabbbbhaijjjm"));
console.log(printerErrors("aaaxbbbbyyhwawiwjjjwwm"));

File: Ruby/printer_errors.rb
def printer_error(s)
	arr = s.split('')
	res = 0
	for c in arr
		if !/[a-m]/.match(c)
			res += 1
		end
	end
	return "#{res}/#{s.length}"
end

p printer_error("aaabbbbhaijjjm")
p printer_error("aaaxbbbbyyhwawiwjjjwwm")